modules/user-directory/Shortcode.php

```
@@ -0,0 +1,581 @@
+        $current_page = 1;
+
+        // Support both ?page=2 and /page/2/
+        if ( get_query_var( 'paged' ) ) {
+            $current_page = (int) get_query_var( 'paged' );
+        } elseif ( ! empty( $_GET['page'] ) ) {
+            // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+            $current_page = (int) $_GET['page'];
+        }
+
+        return max( 1, $current_page );
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'get_current_page'. (undefined)

(UnusedPrivateMethod)